lib/app/routes/exercises.rb

```diff
@@ -133,6 +134,26 @@ class Exercises < Core
         Hack::UpdatesUserExercise.new(submission.user_id, submission.track_id, submission.slug).update
         redirect "/"
       end
+
+      get '/submissions/:key/blobs/:sha' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.